src/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.block {
+  /* width: 96px;
+  height: 96px; */
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  margin: auto;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+}
+
+.stars__star--active {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+/* .stars--0 .stars__star:nth-child(1),
+.stars--0 .stars__star:nth-child(2),
+.stars--0 .stars__star:nth-child(3),
+.stars--0 .stars__star:nth-child(4),
+.stars--0 .stars__star:nth-child(5),
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5),
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5),
+.stars--3 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(5),
+.stars--4 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(3),
+.stars--5 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(5) {
+  background-image: url(images/star-active.svg);
+} */
```

Comment:
try to make this way,  you can use `.stars--5 > .stars__star:nth-child(-n + 5)`

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

here is a good list of selector, could be helpful 😉 
https://www.w3schools.com/cssref/css_selectors.php